QuickFIXn/QuickFix.csproj

```diff
@@ -43,4 +43,8 @@
       <_Parameter1>UnitTests</_Parameter1>
     </AssemblyAttribute>
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.0" />
```

Comment:
So I initially argued to use 6.0.0 because at the time 8.0.2 was pulling in DiagnosticSource unnecessarily. They have since fixed that in 8.0.3, so if it were up to me I would use 8.0.3, but ultimately it does not matter since it is mainly just interface definitions

https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/6.0.0#dependencies-body-tab
https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/8.0.2#dependencies-body-tab
https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/8.0.3#dependencies-body-tab